deploy/crd/crd-v1alpha2.yaml

```diff
@@ -23,6 +23,8 @@ spec:
             slotsPerWorker:
               type: integer
               minimum: 1
+            jobPriority:
+              type: string
```

Comment:
Why is this a string? Shouldn't it be an integer?

Reply:
Hi rongou, jobPriority refers to PriorityClassName just like "system-node-critical" or any other priority.  That is consistent with PriorityClassName in PodGroupSpec of kube-batch scheduling.